Sim2dist

AWBroscius opened the pull request:
Implementation in python of the conversion methods for similarity coefficients from [sim2diss](https://rdrr.io/cran/smacof/man/sim2diss.html)

FarnazH reviewed (changes requested):
This functionality should be implemented in the `converter.py` module.

FanwangM reviewed (changes requested):
This is a big improvement and I enjoy reading the codes a lot. Nice job!

Sorry for being so picky, but we all want to make the package nice and easy to read/maintain.

AWBroscius commented on DiverseSelector/converter.py:
```diff
@@ -0,0 +1,420 @@
+
+"""Utils module."""
+
+import numpy as np
```
I am not sure what you want changed here. Importing `numpy` automatically imports `ndarray`.

FarnazH replied:
The `np.ndarray` is used, so there is no need to import `ndarray`.

AWBroscius commented:
Hi @FanwangM I just pushed another commit that addresses most of your comments. Thank you for the advice. Feel free to look it over again when you have the chance.

FanwangM commented:
I am good with current PR. Do you have any other comments for this PR?  @FarnazH

FarnazH reviewed (approved):
Thanks, @AWBroscius for your contribution. The code looks good to me overall, but I will most likely add some changes later to make it more consistent with the rest of the code.
